cmd/go-getter/main.go

```diff
@@ -56,6 +58,13 @@ func main() {
 	wg := sync.WaitGroup{}
 	wg.Add(1)
 
+	client := getter.DefaultClient
+
+	getters := getter.Getters
```

Comment:
This looks like a change from the current behavior, where callers can define its own list of getters (`client.Getters`) and only those clients get set (originally in `client_options.go`, when it iterates over the getters if not nil). Am I misreading this change? 

Here's an example from terraform: https://github.com/hashicorp/terraform/blob/807267d1b56aa6298fecf5edba98c088c2866e83/internal/initwd/getter.go#L131

Reply:
On the `v2` branch the client was split from the request and the getters are set on the client, so here this line from terraform ([link](https://github.com/hashicorp/terraform/blob/807267d1b56aa6298fecf5edba98c088c2866e83/internal/initwd/getter.go#L50-L58)) could become:

```go
var goGetterClient = getter.Client{
	Getters: map[string]getter.Getter{
		"file":  new(getter.FileGetter),
		"gcs":   new(getter.GCSGetter),
		"git":   new(getter.GitGetter),
		"hg":    new(getter.HgGetter),
		"s3":    new(getter.S3Getter),
		"http":  getterHTTPGetter,
		"https": getterHTTPGetter,
	},
}
```

Reply:
And the call would be : `client.Get(ctx, request)`

Reply:
I see it now, thank you! Definitely a misread on my part.